manifest_staging/chart/gatekeeper-operator/helm-modifications/helm-modifications.yaml

```diff
@@ -54,6 +54,7 @@ spec:
             - --logtostderr
             - --constraint-violations-limit={{ .Values.constraintViolationsLimit }}
             - --audit-from-cache={{ .Values.auditFromCache }}
+            - --exempt-namespaces=gatekeeper-system
```

Comment:
nit: update this based on your recent code change

Reply:
updated